src/rgw/rgw_op.cc

```diff
@@ -5585,8 +5586,8 @@ void RGWDeleteCORS::execute(optional_yield y)
     });
 }
 
-void RGWOptionsCORS::get_response_params(string& hdrs, string& exp_hdrs, unsigned *max_age) {
-  get_cors_response_headers(rule, req_hdrs, hdrs, exp_hdrs, max_age);
+void RGWOptionsCORS::get_response_params(const DoutPrefixProvider *dpp, string& hdrs, string& exp_hdrs, unsigned *max_age) {
```

Comment:
`RGWOptionsCORS` inherits from `RGWOp` so it is a dpp. No need to change this function call and you can use `this` below.